DependencyInjection/Configuration.php

```diff
@@ -57,7 +62,7 @@ public function getConfigTreeBuilder()
                 ->arrayNode('param_fetcher_listener')
                     ->beforeNormalization()
                         ->ifString()
-                        ->then(function ($v) { return array('enabled' => in_array($v, array('force', 'true')), 'force' => 'force' === $v); })
+                        ->then(function ($v) { return ['enabled' => in_array($v, ['force', 'true']), 'force' => 'force' === $v]; })
```

Comment:
no, it is being used - i'm saying the fixes were applied BEFORE the config was commited